### PR TITLE
add quest Shadows of the Departed

### PR DESCRIPTION
**Pull request by KnowOne134**

### CODE REVIEW

File: scripts/zones/Norg/Zone.lua [hunk 1]
```diff
@@ -4,54 +4,49 @@
...
+    if player:getXPos() == 0 and player:getYPos() == 0 and player:getZPos() == 0 then
+        player:setPos(-19.238,-2.163,-63.964,187)
     end
-    if (player:getCurrentMission(ZILART) == dsp.mission.id.zilart.THE_NEW_FRONTIER) then
-        cs = 1;
-    elseif (player:getCurrentMission(ZILART) == dsp.mission.id.zilart.AWAKENING and player:getVar("ZilartStatus") == 0 or player:getVar("ZilartStatus") == 2) then
-        cs = 176;
+    if player:getCurrentMission(ZILART) == dsp.mission.id.zilart.THE_NEW_FRONTIER then
+        cs = 1
+    elseif player:getCurrentMission(ZILART) == dsp.mission.id.zilart.AWAKENING and player:getVar("Awakening_Norg") == 1 then
```
Comment (TeoTwawki) [inline scripts/zones/Norg/Zone.lua]: why are you changing the variable name and value for a long standing working mission.

File: scripts/zones/Promyvion-Dem/npcs/_0id.lua [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+--  Area: Promyvion Dem
+--   NPC: Memory Flux
+--  !pos 220 0 340 18
+-----------------------------------
+require("scripts/globals/npc_util")
+-----------------------------------
+
+function onTrigger(player,npc)
+    if player:getQuestStatus(JEUNO,dsp.quest.id.jeuno.SHADOWS_OF_THE_DEPARTED) == QUEST_ACCEPTED and not player:hasKeyItem(dsp.ki.PROMYVION_DEM_SLIVER) then 
+		npcUtil.giveKeyItem(player, dsp.ki.PROMYVION_DEM_SLIVER)
```
Comment (TeoTwawki) [inline scripts/zones/Promyvion-Dem/npcs/_0id.lua]: tabs

File: scripts/zones/Promyvion-Mea/npcs/_0k0.lua [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+--  Area: Promyvion Mea
+--   NPC: Memory Flux
+--  !pos 259 0 179 20
+-----------------------------------
+require("scripts/globals/npc_util")
+-----------------------------------
+
+function onTrigger(player,npc)
+    if player:getQuestStatus(JEUNO,dsp.quest.id.jeuno.SHADOWS_OF_THE_DEPARTED) == QUEST_ACCEPTED and not player:hasKeyItem(dsp.ki.PROMYVION_MEA_SLIVER) then 
+		npcUtil.giveKeyItem(player, dsp.ki.PROMYVION_MEA_SLIVER)
```
Comment (TeoTwawki) [inline scripts/zones/Promyvion-Mea/npcs/_0k0.lua]: tabs

**Comment by TeoTwawki** [comment]
This is really annoying to try and review because of the many format corrections included in the same commit as the actual code changes.
every;
line;
(dude);
is;
highlight;

File: scripts/zones/Norg/Zone.lua (hunk 1 above)
Comment (KnowOne134) [reply scripts/zones/Norg/Zone.lua]: seemed much simpler check to check 2 cs's in any order

File: scripts/zones/Promyvion-Dem/npcs/_0id.lua (hunk 2 above)
Comment (KnowOne134) [reply scripts/zones/Promyvion-Dem/npcs/_0id.lua]: my copy paste from old scripts, blah

File: scripts/zones/Norg/Zone.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/zones/Norg/Zone.lua]: it worked just fine with ZilartStatus and convention has always been expansion+status for the variable name on missions. That isn't changing without a serious discussion.

**Comment by KnowOne134** [comment]
I see it now. End result is 3. I'll change those parts back. 1 has addvar 1
and other is addvar  2

**Comment by zynjec** [comment]
Needs to be rebased.

**Pull request closed without merging**